### PR TITLE
refactor

### PR DESCRIPTION
**Pull request by ZingBallyhoo**
cleanup

.. i guess

### CODE REVIEW

File: src/DungeonTools.Save.File/DungeonTools.Save.File.csproj [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
     <PropertyGroup>
         <Description>Processing of encrypted data for DungeonTools.</Description>
+        <RootNamespace>DungeonTools.Encryption</RootNamespace>
```
Comment (ZingBallyhoo) [inline src/DungeonTools.Save.File/DungeonTools.Save.File.csproj:6]: what

File: src/DungeonTools.Cli/Program.cs [hunk 2]
```diff
@@ -5,84 +5,70 @@
...
             }
 
-            string outputFile = GetOutputFilePath(file, inputType == DataType.UnsafeEncrypted, overwrite);
+            processed.Position = 0;
+            string outputFile = GetOutputFilePath(file, encrypted, overwrite);
             await using FileStream outputStream = File.Open(outputFile, FileMode.Create, FileAccess.Write);
             await processed.CopyToAsync(outputStream);
         }
 
         [SuppressMessage("ReSharper", "ConvertIfStatementToSwitchStatement")]
```
Comment (HellPie) [inline src/DungeonTools.Cli/Program.cs:52]: SuppressMessage is not needed anymore

File: src/DungeonTools.Save.Models/DungeonTools.Save.Models.csproj [hunk 3]
```diff
@@ -3,5 +3,6 @@
 
     <PropertyGroup>
         <Description>Data classes and serialization for DungeonTools.</Description>
+        <RootNamespace>DungeonTools.SaveFiles</RootNamespace>
```
Comment (HellPie) [inline src/DungeonTools.Save.Models/DungeonTools.Save.Models.csproj]: Wrong namespace

File: src/DungeonTools.Save.File/AesEncryptionService.NoKeys.cs [hunk 4]
```diff
@@ -6,23 +6,18 @@
...
+// Please store those in a file named "AesEncryptionProvider.Keys.cs" surrounding it with an "#ifdef USE_KEYS"
 // macro block so that this file will not clash with it during compile time.
 // Using a different file in a partial class ending in ".Keys.cs" triggers a .gitignore rule so encryption
 // keys are never committed with the project's sources.
 
-// The creation of a file named "AesEncryptionService.Keys.cs" containing only an Algorithm property with a set IV
-// and Key is the only exception to this project's license (AGPL-3.0).
-
-using System.Security.Cryptography;
+// The creation of a file named "AesEncryptionProvider.Keys.cs" containing GetKey and GetIV methods is the only exception to this project's license (AGPL-3.0).
```
[the remark this answers is not in the document]
Comment (HellPie) [reply src/DungeonTools.Save.File/AesEncryptionService.NoKeys.cs:14]: I'll fix this myself with the syntax stuff

File: src/DungeonTools.Save.Models/DungeonTools.Save.Models.csproj (hunk 3 above)
Comment (ZingBallyhoo) [reply src/DungeonTools.Save.Models/DungeonTools.Save.Models.csproj]: look at latest source then maybe

**Pull request merged**